core/templates/pages/exploration-editor-page/services/version-history.service.ts

```diff
@@ -153,43 +157,52 @@ export class VersionHistoryService {
   }
 
   getBackwardStateDiffData(): StateDiffData {
+    // The following (this.currentPositionInStateVersionHistoryList || 0) is
+    // done to fix the typescript errors. Otherwise, when this function will
```

Comment:
"This is done to fix the typescript errors" isn't a sufficient justification.

The explanation for a default value needs to refer to what conceptually is going on. Could you try to explain this again?

Reply:
I have removed this now.